error compiling vdr-addon

libtool: compile:  /home/me/development/OpenELEC.tv/build.OpenELEC_PVR-Fusion.x86_64-devel/toolchain/bin/x86_64-openelec-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I.. -I/usr/include/python2.7 -I/home/me/development/OpenELEC.tv/build.OpenELEC_PVR-Fusion.x86_64-devel/toolchain/x86_64-openelec-linux-gnu/sysroot/usr/include/libxml2 -I../libxslt -I.. -I../libexslt -march=btver1 -m64 -fomit-frame-pointer -Wall -pipe -O3 -fexcess-precision=fast -ffast-math -fPIC -Wall -MT types.lo -MD -MP -MF .deps/types.Tpo -c types.c  -fPIC -DPIC -o .libs/types.o
In file included from /usr/include/python2.7/Python.h:58:0,
                 from libxml_wrap.h:1,
                 from types.c:13:
/usr/include/python2.7/pyport.h:849:2: error: #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."
In file included from /usr/include/python2.7/Python.h:58:0,
                 from libxslt.c:14:
/usr/include/python2.7/pyport.h:849:2: error: #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."
In file included from /usr/include/python2.7/Python.h:58:0,
                 from libxml_wrap.h:1,
                 from libxslt_wrap.h:1,
                 from libxslt-py.c:4:
/usr/include/python2.7/pyport.h:849:2: error: #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."
make[3]: **\* [types.lo] Error 1
make[3]: **\* Waiting for unfinished jobs....
libxslt.c: In function 'initlibxsltmod':
libxslt.c:1183:15: warning: variable 'm' set but not used [-Wunused-but-set-variable]
make[3]: **\* [libxslt.lo] Error 1
make[3]: **\* [libxslt-py.lo] Error 1

maybe because it is including host python from  -I/usr/include/python2.7

--- a/packages/3rdparty/lib/libxslt/build
+++ b/packages/3rdparty/lib/libxslt/build
@@ -36,6 +36,7 @@ fi
              --enable-shared \
              $CONFIG_LIBXML \
              --disable-ipv6 \
+             --without-python \
              --without-crypto
 
 make

